import_and_export_data/formats/attribute.rst

```diff
@@ -10,7 +10,7 @@ The import/export file follows this data structure:
     :file: examples/attribute_properties.csv
     :delim: ;
 
-.. _MeasureBundle measures: https://github.com/akeneo/MeasureBundle/blob/master/Resources/config/measure.yml
+.. _MeasureBundle measures: https://github.com/akeneo/MeasureBundle/blob/3.0/Resources/config/measure.yml
```

Comment:
broken link

Reply:
Yep, the check of the hyperlink by Travis was removed this summer :disappointed:. We will put it back.

Reply:
Yes for now ;)